packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -251,7 +251,7 @@ const _StyledBaseInput: React.ForwardRefRenderFunction<
     isFocused: currentInteraction === 'active',
   };
 
-  return hasPopup ? (
+  return renderAs === 'button' ? (
```

Comment:
logic change? Won't it affect popup?

Reply:
I used popup terminology to refer to dropdown earlier 🙈 So it was always rendering button in Dropdown. But with AutoComplete, our deciding factor is this renderAs prop. We render button only when renderAs is button. Earlier we didn't have input component + dropdown together so this logic worked fine.